ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyflie.py

```diff
@@ -96,26 +215,104 @@ def uploadTrajectory(self, trajectoryId, pieceOffset, trajectory):
         self.uploadTrajectoryService(trajectoryId, pieceOffset, pieces)
 
     def startTrajectory(self, trajectoryId, timescale = 1.0, reverse = False, relative = True, groupMask = 0):
+        """Begins executing a previously uploaded trajectory.
+
+        Asynchronous command; returns immediately.
+
+        Args:
+            trajectoryId (int): ID number as given to :meth:`uploadTrajectory()`.
+            timescale (float): Scales the trajectory duration by this factor.
+                For example if timescale == 2.0, the trajectory will take twice
+                as long to execute as the nominal duration.
+            reverse (bool): If true, executes the trajectory backwards in time.
+            relative (bool): If true (default), the position of the trajectory
+                is shifted such that it begins at the current position setpoint.
+                This is usually the desired behavior.
+            groupMask (int): Group mask bits. See :meth:`setGroupMask()` doc.
+        """
         self.startTrajectoryService(groupMask, trajectoryId, timescale, reverse, relative)
 
     def position(self):
+        """Returns the true position as determined by motion capture.
+
+        Blocking command. Waits for a fresh update from the mocap system.
+        TODO: make a non-blocking version that potentially gives stale data.
```

Comment:
This is actually exactly what the function is doing already: waitForTransform with a time of zero simply returns if at least one transform is in the tree. Thus, this function is only blocking at the beginning, when there was never any information from the Mocap.

Reply:
Fixed.